toggles/webapp/toggles.ts

```diff
@@ -42,7 +41,14 @@ const toggles = {
       description: 'A toolbar to help us navigate the secret depths of the API',
     },
   ] as const,
-  tests: [] as ABTest[],
+  tests: [
+    {
+      id: 'gaSecureCookies',
+      title: 'GA Secure Cookies',
+      range: [0, 10],
```

Comment:
Maybe put a comment here about how you picked the test size?

Reply:
What is a reasonable amount of users to test this with do we think? (ping @taceybadgerbrook). Ten percent was a number mentioned in a stand-up I think, and I rolled with it because it sounded sensible, but didn't give it any more thought. I guess it needs to be large enough to show us that it's working in a reasonable amount of time, and small enough that if it _does_ break anything, we don't lose too much data.

Reply:
IMO it's fine if the comment is "I picked this somewhat arbitrarily", because that's more useful than leaving it a mystery – the reader knows there isn't some secret rationale.

Reply:
@davidpmccormick reading [this article again](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie/SameSite), my understanding is that SameSite is a setting that all Chrome, Edge, Firefox, Opera and Safari recognise and only recently have they required SameSite=secure before sending cookies. The difference in setting SameSite=none and secure is in improved security for [wc.org](http://wc.org/). It has nothing to do with potentially personally identifiable data.

Is this correct?

Reply:
Correct, it doesn't have to do with personally identifiable data. We're making the change in order that GA continues to work.

Reply:
So far from user numbers being negatively impacted, we could potentially see an _increase_ in user numbers because previously no cookies were sent...